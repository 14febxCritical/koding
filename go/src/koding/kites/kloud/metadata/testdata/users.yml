#cloud-config
hostname: 'user'

users:
  - name: 'otheruser'
    lock_passwd: True
    gecos: Koding
    groups: ['sudo']
    sudo: ['ALL=(ALL) NOPASSWD:ALL']
    shell: /bin/bash
