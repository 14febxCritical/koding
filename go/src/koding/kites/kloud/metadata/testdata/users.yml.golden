#cloud-config
hostname: user
output:
  all: '| tee -a /var/log/cloud-init-output.log'
runcmd:
- /var/lib/koding/provision.sh
users:
- gecos: Koding
  groups:
  - sudo
  lock_passwd: true
  name: otheruser
  shell: /bin/bash
  sudo:
  - ALL=(ALL) NOPASSWD:ALL
- default
- gecos: Koding
  groups:
  - sudo
  lock_passwd: true
  name: johndoe
  shell: /bin/bash
  sudo:
  - ALL=(ALL) NOPASSWD:ALL
write_files:
- content: kitekey
  path: /etc/kite/kite.key
- content: metadata
  path: /var/lib/koding/metadata.json
  permissions: "0644"
- content: userdata
  encoding: b64
  path: /var/lib/koding/user-data.sh
  permissions: "0755"
- content: provision
  path: /var/lib/koding/provision.sh
  permissions: "0755"
